spec/factories/poems.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :poem do
+    title "hogehoge"
+    description "fugafuga"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)